src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
Here you are using "Colors.values()" twice, so you can create a variable for it.
```suggestion
        Colors[] colors = Colors.values();
        int index = new Random().nextInt(colors.length);
        return colors[index].toString();
```